### ISSUE TITLE
Cannot access secondary GPU - error: Could not load GPU driver

### ISSUE THREAD

**Issue by NamesAreUseless** [Bumblebee-Project/Bumblebee#736]
When I originally installed bumblebee, its worked just fine (http://community.linuxmint.com/tutorial/view/1961).  Its only just recently that now that when I try to run "optirun APP" (APP being an application), I receive the following error:

`[ERROR]Cannot access secondary GPU - error: Could not load GPU driver`
`[ERROR]Aborting because fallback start is disabled.`

I am running Linux Mint 17.3 Rosa, Cinnamon 64-bit

Already attempted these steps, still same error: http://ubuntuforums.org/showthread.php?t=2043971&p=12179206#post12179206

Anyone suppose they can help me out?

**Comment by getsmp** [comment]
Possible duplicate of #604. You might first read that.

**Comment by NamesAreUseless** [comment]
Similar problem yes, but when I output `modinfo nvidia-current` I receive:
`modinfo: ERROR: Module nvidia-current not found.`

And when I output `modinfo nvidia-uvm` I receive:
`modinfo: ERROR: Module nvidia-uvm not found.`

Let me list my LibraryPath and XorgModulePath like that user did:
`# colon-separated path to the nvidia libraries
LibraryPath=/usr/lib/nvidia-current:/usr/lib32/nvidia-current
# comma-separated path of the directory containing nvidia_drv.so and the
# default Xorg modules path
XorgModulePath=/usr/lib/nvidia-current/xorg,/usr/lib/xorg/modules
XorgConfFile=/etc/bumblebee/xorg.conf.nvidia`

So unfortunately, it did not help. Here are my system specs output with `uname -a`:
`Linux matt-GE72-2QF 3.19.0-32-generic #37~14.04.1-Ubuntu SMP Thu Oct 22 09:41:40 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by NamesAreUseless** [comment]
So I think I've fixed my problem. This is going to sound silly, but the solution to the problem was opening Driver Manager in Linux Mint, and selecting the correct driver.

Previously I was using "xserver-xorg-video-noveau". I've now selected "nvidia-352" (the latest and greatest for my Nvidia GTX970M I believe), and viola! "optirun APP" works just fine, and my programs are running a lot more efficiently now.

So yes, what was a pain-staking problem had a very easy solution.
